book/09-git-and-other-scms/1-git-and-other-scms.asc

```diff
@@ -12,7 +12,7 @@ The second part of this chapter covers how to migrate your project into Git from
 
 (((Git as a client)))
 Git provides such a nice experience for developers that many people have figured out how to use it on their workstation, even if the rest of their team is using an entirely different VCS.
-There are a number of these adapters, called ``bridges,'' available.
+There are a number of these adapters, called ``bridges'', available.
```

Comment:
I had a feeling about this, and after some research it seems I was correct: the comma goes inside the quotation marks.

[The Punctuation Guide](http://www.thepunctuationguide.com/quotation-marks.html#adjacentpunctuation):

> Commas and periods that are part of the overall sentence go inside the quotation marks, even though they aren’t part of the original quotation.

[GrammarBook.com](http://www.grammarbook.com/punctuation/quotes.asp):

> Rule 4. Periods and commas ALWAYS go inside quotation marks.